test/scenarios/driver/docker/molecule/default/tests/test_default.py

```diff
@@ -30,4 +30,4 @@ def test_etc_molecule_ansible_hostname_file(host):
 
 def test_buildarg_env_var(host):
     cmd_out = host.run("echo $envarg")
-    assert cmd_out.stdout == 'this_is_a_test'
+    assert 'this_is_a_test' in cmd_out.stdout
```

Comment:
Why `in`?

Reply:
The test was failing with a missing `\n` in the exact string match. I'd rather not have tests failing with such brittle checks, so I change it to a `in` check. This will be more resilient to any other unimportant changes to string handling and we still get a useful test that checks for what we want.

Reply:
I'd add that `\n` there then. Or do `str.strip()`. Exact checks are more reliable IMHO.

Reply:
Done.